components/director/pkg/formationconstraint/validation.go

```diff
@@ -198,6 +200,33 @@ func emptyNotificationStatusReturnedOperationDetails() *NotificationStatusReturn
 			Value: json.RawMessage("\"\""),
 			Error: json.RawMessage("\"\""),
 		},
+		FormationAssignmentTemplateInput: &webhook.ApplicationTenantMappingInput{
+			Operation:   model.AssignFormation,
+			FormationID: "",
+			Formation: &model.Formation{
+				ID:                  "",
+				FormationTemplateID: "",
+				Name:                "",
```

Comment:
We could skip these empty strings as the zero/default value is `""` to minimize the size of the structure. Valid for `FormationID` field(+ other fields) as well. e.g. fixApplicationModel -> BaseEntity